core/src/sigverify.rs

```diff
@@ -249,14 +287,18 @@ pub fn ed25519_verify(
     let mut num = 0;
     for (vs, sig_vs) in rvs.iter_mut().zip(sig_lens.iter()) {
         for (v, sig_v) in vs.iter_mut().zip(sig_vs.iter()) {
-            let mut vout = 1;
-            for _ in 0..*sig_v {
-                if 0 == out[num] {
-                    vout = 0;
+            if *sig_v == 0 {
```

Comment:
As mentioned in the pull request description, this part changes the runtime semantics. I'm assuming there will be no transaction with zero signature...